lib/ansible/modules/network/fortios/fortios_antivirus_heuristic.py

```diff
@@ -44,37 +41,48 @@
     - fortiosapi>=0.9.8
 options:
     host:
-       description:
-            - FortiOS or FortiGate ip address.
-       required: true
+        description:
+            - FortiOS or FortiGate IP address.
+        type: str
+        required: true
     username:
         description:
             - FortiOS or FortiGate username.
+        type: str
         required: true
```

Comment:
Same as above.

Reply:
Good spot. I'll fix it and submit a new commit